cpg_workflows/jobs/stripy.py

```diff
@@ -23,7 +23,7 @@ def stripy(
     json_path: Path,
     analysis_type: str = 'standard',
     job_attrs: dict | None = None,
-    overwrite: bool = False,
+    overwrite: bool = False
```

Comment:
Is removal of comma intentional?

Reply:
No. just an artefact of merges that I did not bother to revert.  I can add it back in.